doc/registry_api.md

```diff
@@ -0,0 +1,67 @@
+# Suggested API for registries
+
+```ruby
+# Returns the component matching the parameters from the registry or nil if
+# the component is not available. If multiple versions of the component is matching
+# the component with the highest version is returned.
+#
+# @param [String] component
+# @param [nil|String|Gem::Requirement|Gem::Version] version
+# @return [Diversity::Component|nil]
+# @alias get
```

Comment:
I'm sorry, but that's not very helpful at all... :smirk: 

The registry might **previously** have fetched the data, making it "accessing what is already there"...or it might not, in which case we will get a new instance...

I really don't mind aliases, but if we should have just one method, I'm voting for using __get__. It's somewhat more descriptive and having to type one more character each time does not really hurt to much. And besides, it's ruby...people can create thir own aliases anytime.

Is using **get** ok with you?